DeepSpeech.ipynb

```diff
@@ -446,7 +446,7 @@
     "    lstm_bw_cell = tf.nn.rnn_cell.BasicLSTMCell(n_cell_dim, forget_bias=1.0)\n",
     "    \n",
     "    # Split data because rnn cell needs a list of inputs for the BRNN inner loop\n",
-    "    layer_3 = tf.split(0, n_steps, layer_3)\n",
```

Comment:
No, but isn't that the shape we want it to be after it gets split (not saying necessarily that it is, am asking so I know)? It's ok that we have n_steps, but not ok that it isn't a tensor - if we can derive it from the input, that's great though.